hooks/scripts/cfn_lint.sh

```diff
@@ -11,7 +11,7 @@ set -e
 
 CF_CHECK='AWSTemplateFormatVersion'
 current_commit=$(git rev-parse HEAD)
-templates=$(find `pwd` -type f -exec grep -l $CF_CHECK {} \;)
+templates=$(find `pwd` -type f -exec grep -l $CF_CHECK --include \*.yaml --include \*.yml --include \*.json {} \;)
```

Comment:
Personally, I'd move the file name filtering to the `find` command instead of the `grep` command, but I'm not sure it makes much of a difference.

Is `CF_CHECK` used anywhere else? There doesn't seem much use in creating a variable that will be used only once on a line almost immediately following.

Reply:
@sirhc Thank you for pointing it out. It would have more or less the same effect here I guess.

`CF_CHECK` is used only here. Keeping it since I plan to raise a separate discussion/PR around overhauling `pre-commit`scripts.